.github/workflows/backend.yml

```diff
@@ -22,3 +22,15 @@ jobs:
     - name: Build release and call executable
       working-directory: ./backend
       run: cargo run --release -- --help
+
+    - name: Build and Push template image
+      uses: docker/build-push-action@v1 # https://github.com/docker/build-push-action
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        dockerfile: Dockerfile
+        repository: paritytech/substrate-telemetry-backend-${{ github.event.client_payload.id }}
+        tags: latest
+        add_git_labels: true
+        tag_with_ref: true
+        tag_with_sha: true
```

Comment:
Check on [this](https://github.com/docker/build-push-action#tag_with_ref) page if you actually need to specify the `latest` tag - from looking at it, it seems to say that when `master` is pushed (and `tag_with_ref` is set to true), the image will automatically be tagged 'latest'.

Reply:
I want to make sure that the tag this pushes to actually never differs, because I'll be watching a single tag for updates. Maybe i should remove tag_with_ref and tag_with_sha, then. the name of the tag is not set in stone.